p2p/nat/check.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2021 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package nat
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/mysteriumnetwork/node/core/port"
+)
+
+const (
+	reqTimeout   = 2 * time.Second
+	totalTimeout = 5 * time.Second
+)
+
+type checkResp struct {
+	ok  bool
+	err error
+}
+
+func checkAllPorts(ports []int) error {
+	var addresses []string
+
+	for _, address := range strings.Split(config.GetString(config.FlagPortCheckServers), ",") {
+		address = strings.TrimSpace(address)
+		if address != "" {
+			addresses = append(addresses, address)
+		}
+	}
+
```

Comment:
Add a check to see if you even have any addresses to use? Not point to continue if not.

Reply:
We already have this check in a level deeper.